### PR TITLE
Migration to tailwind

### PR DESCRIPTION
**Pull request by bejoinka**
This one touches a few files... made some changes to try and make it easier to work with:
1. using daisyui and tailwind instead of ui-library and css/scss. i left ui-library as a dev dependency for now, but removed the `import '@awell-health/ui-library/dist/styles.css'` in the storybook preview, as i believe it's no longer necessary... if @nckhell thinks we can remove it entirely from the repo, i'm great with that, too.
2. replaced the lodash package with lodash-es (we should do that everywhere, perhaps -- tree-shakeable -- the alternative is `import isNil from 'lodash/isNil`, which requires one line per function... i opened for lodash-es in this project)
3. added a plugin (`rollup-plugin-visualizer`) that helps us to understand what is contributing to package size. it creates a `stats.html` file that can be opened in a browser window. super helpful to figuring out what was causing the package to be so large.
4. removed the FullCalendar, as it's not necessary

Separately from the refactor, i added a bit of responsive behavior into the scheduling component (mostly in the WeekCalendar component) so it would look better on mobile

what's left (outside the scope of this MR)?
- [ ] a cleanup in some of the codebase to make it easier to read
- [ ] the rest of the work

i'll start with the search filter today if i have some time.

### CODE REVIEW

File: src/hostedPages/SchedulingActivity.tsx [hunk 1]
```diff
@@ -110,6 +109,7 @@ export const SchedulingActivity: FC<SchedulingActivityProps> = ({
     (date: Date) => {
       onDateSelect(date);
 
+      console.log('onDateSelect', date);
```
Comment (sharlotta93) [inline src/hostedPages/SchedulingActivity.tsx:112]: was this left on purpose?

**Review by sharlotta93 — APPROVED** [review]

File: src/hostedPages/SchedulingActivity.tsx (hunk 1 above)
Comment (bejoinka) [reply src/hostedPages/SchedulingActivity.tsx:112]: nope, i should remove!

File: src/hostedPages/SchedulingActivity.tsx [hunk 2]
```diff
@@ -120,6 +120,7 @@ export const SchedulingActivity: FC<SchedulingActivityProps> = ({
     (slot: SlotType) => {
       onSlotSelect(slot);
 
+      console.log('onSlotSelect', slot);
```
[the remark this answers is not in the document]
Comment (bejoinka) [reply src/hostedPages/SchedulingActivity.tsx:123]: ditto, gotta remove!

File: src/atoms/BookingConfirmation/BookingConfirmation.tsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { FC } from 'react';
```
Comment (nckhell) [inline src/atoms/BookingConfirmation/BookingConfirmation.tsx:1]: @bejoinka @sharlotta93 Note: this component/stage is not used anymore in the Scheduling flow as showing confirmation of the booking happens now in a Message action. So in theory this component could also be removed.

File: src/atoms/Calendar/week/WeekCalendar.tsx [hunk 4]
```diff
@@ -130,50 +117,80 @@ export const WeekCalendar: FC<WeekCalendarProps> = ({
     weekStartsOn
   ]);
 
+  const chevronClasses = clsx(
+    'flex flex-none align-center justify-center',
+    'text-primary size-8 font-bold'
+  );
+
+  const cannotActivate = (day: (typeof days)[0]) => {
```
Comment (nckhell) [inline src/atoms/Calendar/week/WeekCalendar.tsx:125]: @bejoinka @sharlotta93 This could use a useMemo as this variable now be computed on every re-render.

**Pull request merged**